bmlt-cms-satellite-plugin.php

```diff
@@ -30,9 +30,10 @@
 
 // Include the satellite driver class.
 if (!defined('ROOTPATH')) {
-    define('ROOTPATH', __DIR__);
+    require_once ( __DIR__.'/../bmlt-satellite-driver/bmlt_satellite_controller.class.php' );
```

Comment:
theres nothing that is nor any situation that i know of where this could happen, however im confused why we can't just do this here.
`require_once ( __DIR__.'/../bmlt-satellite-driver/bmlt_satellite_controller.class.php' ); `

Reply:
Because that’s is not the correct location of the dependency in the root server.

Reply:
ahh true true

Reply:
You should validate that that’s true.

Reply:
yea it is